core-examples/src/main/java/io/vertx/example/core/ha/Server.java

```diff
@@ -0,0 +1,22 @@
+package io.vertx.example.core.ha;
+
+import io.vertx.core.AbstractVerticle;
+
+import java.lang.management.ManagementFactory;
+
+/*
+ * This is just a simple verticle creating a HTTP server. The served response contains an id identifying the process
+ * for illustration purpose as it will change when the verticle is migrated.
+ *
+ * The verticle is intended to be launched using the `-ha` option.
+ */
+public class Server extends AbstractVerticle {
+
```

Comment:
If you add a main() method the deploys the verticle then this could be run in the IDE. Many users won't have a vert.x installed so running from the command line might be a pain.